src/core/stdc/errno.d

```diff
@@ -1000,6 +1000,138 @@ else version( FreeBSD )
+
+    /* should be rearranged */
+    enum EHOSTDOWN       = 64              /** Host is down */;
+    enum EHOSTUNREACH    = 65              /** No route to host */;
+    enum ENOTEMPTY       = 66              /** Directory not empty */;
+
+    /* quotas & mush */
+    enum EPROCLIM        = 67              /** Too many processes */;
+    enum EUSERS          = 68              /** Too many users */;
+    enum EDQUOT          = 69              /** Disc quota exceeded */;
+
+    /* Network File System */
+    enum ESTALE          = 70              /** Stale NFS file handle */;
+    enum EREMOTE         = 71              /** Too many levels of remote in path */;
+    enum EBADRPC         = 72              /** RPC struct is bad */;
+    enum ERPCMISMATCH    = 73              /** RPC version wrong */;
+    enum EPROGUNAVAIL    = 74              /** RPC prog. not avail */;
+    enum EPROGMISMATCH   = 75              /** Program version wrong */;
+    enum EPROCUNAVAIL    = 76              /** Bad procedure for program */;
+
+    enum ENOLCK          = 77              /** No locks available */;
+    enum ENOSYS          = 78              /** Function not implemented */;
+
+    enum EFTYPE          = 79              /** Inappropriate file type or format */;
+    enum EAUTH           = 80              /** Authentication error */;
+    enum ENEEDAUTH       = 81              /** Need authenticator */;
+
+    /* SystemV IPC */
+    enum EIDRM           = 82              /** Identifier removed */;
+    enum ENOMSG          = 83              /** No message of desired type */;
+    enum EOVERFLOW       = 84              /** Value too large to be stored in data type */;
+    /* Wide/multibyte-character handling, ISO/IEC 9899/AMD1:1995 */
+    enum EILSEQ          = 85              /** Illegal byte sequence */;
+
+    /* From IEEE Std 1003.1-2001 */
+    /* Base, Realtime, Threads or Thread Priority Scheduling option errors */
+    enum ENOTSUP         = 86              /** Not supported */;
+
+    /* Realtime option errors */
+    enum ECANCELED       = 87              /** Operation canceled */;
+
+    /* Realtime, XSI STREAMS option errors */
+    enum EBADMSG         = 88              /** Bad or Corrupt message */;
+
+    /* XSI STREAMS option errors  */
+    enum ENODATA         = 89              /** No message available */;
+    enum ENOSR           = 90              /** No STREAM resources */;
+    enum ENOSTR          = 91              /** Not a STREAM */;
+    enum ETIME           = 92              /** STREAM ioctl timeout */;
+
+    /* File system extended attribute errors */
+    enum ENOATTR         = 93              /** Attribute not found */;
+
+    /* Realtime, XSI STREAMS option errors */
+    enum EMULTIHOP       = 94              /** Multihop attempted */; 
+    enum ENOLINK         = 95              /** Link has been severed */;
+    enum EPROTO          = 96              /** Protocol error */;
+}
```

Comment:
It looks like you got these comments from the official NetBSD header, please replace them with blank comments, as seen for Linux.  Comments are copyrightable, it is Walter's policy not to copy them.